Code/src/main/java/nl/utwente/group10/ui/gestures/InputAnchorHandler.java

```diff
@@ -0,0 +1,129 @@
+package nl.utwente.group10.ui.gestures;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import nl.utwente.group10.ui.CustomUIPane;
+import nl.utwente.group10.ui.components.Connection;
+import nl.utwente.group10.ui.components.ConnectionAnchor;
+import nl.utwente.group10.ui.components.ConnectionLine;
+import nl.utwente.group10.ui.components.InputAnchor;
+import nl.utwente.group10.ui.components.OutputAnchor;
+import javafx.event.EventHandler;
+import javafx.geometry.Point2D;
+import javafx.scene.input.InputEvent;
+import javafx.scene.input.MouseDragEvent;
+import javafx.scene.input.MouseEvent;
+import javafx.scene.input.TouchEvent;
+
+public class InputAnchorHandler implements EventHandler<InputEvent> {
```

Comment:
Nee, zoals besproken op Whatsapp lijkt het me beter als deze structuur strakgetrokken wordt. Hoe touch dan op die nieuwe structuur past zien we later wel.